### PR TITLE
fix #787 : load simulation config at startup

### LINKED ISSUE #787: Bug/simulation: simulation mode seems to not be supported anymore
**Describe the bug**
The simulation mode doesn't seems to work anymore :(

**To Reproduce**
1. enable the simulation mode globally
2. attack your server
3. inspect the alert to check if the alert is in simulation

**Expected behavior**
The alert to be simulated

**Technical Information (please complete the following information):**
 - crowdsec v1.0.13

### PR DESCRIPTION
**Pull request by he2ss**
* load simulation config at startup
* don't send alerts in simulated mode to capi
* add functional tests on simulation

### CODE REVIEW

File: pkg/csconfig/crowdsec_service.go [hunk 1]
```diff
@@ -64,6 +64,9 @@ func (c *Config) LoadCrowdsec() error {
 	if c.Crowdsec.AcquisitionDirPath == "" && c.Crowdsec.AcquisitionFilePath == "" {
 		return fmt.Errorf("no acquisition_path nor acquisition_dir")
 	}
+	if err := c.LoadSimulation(); err != nil {
+		log.Errorf("load error (simulation) : %s", err)
```
Comment (buixor) [inline pkg/csconfig/crowdsec_service.go:68]: can we `errors.Wrap` here plz ? :)

**Review by buixor — APPROVED** [review]
overall LGTM, minor style remark. Before merging I guess we want some functional tests

File: pkg/csconfig/crowdsec_service.go (hunk 1 above)
Comment (he2ss) [reply pkg/csconfig/crowdsec_service.go]: done

**Review by buixor — APPROVED** [review]
LGTM

**Pull request merged**